pkg/deploy/controller/deployment/controller.go

```diff
@@ -83,19 +83,19 @@ func (c *DeploymentController) Handle(deployment *kapi.ReplicationController) er
 					deployment.Annotations[deployapi.DeploymentStatusAnnotation] = failedStatus
 					deployment.Annotations[deployapi.DeploymentStatusReasonAnnotation] = deployapi.DeploymentFailedUnrelatedDeploymentExists
 					if _, err := c.deploymentClient.updateDeployment(deployment.Namespace, deployment); err != nil {
-						c.recorder.Eventf(deployment, "failedUpdate", "Error updating deployment %s status to %s", deployutil.LabelForDeployment(deployment), failedStatus)
-						glog.Errorf("Error updating deployment %s status to %s", deployutil.LabelForDeployment(deployment), failedStatus)
+						c.recorder.Eventf(deployment, "failedUpdate", "Error updating Deployment %s/%s status to %s", deployment.Namespace, deployutil.LabelForDeployment(deployment), failedStatus)
+						glog.Errorf("Error updating Deployment %s/%s status to %s", deployment.Namespace, deployutil.LabelForDeployment(deployment), failedStatus)
```

Comment:
NO!, we only lowercase the ones that go further. This one goes straight to log. I'm wondering if it shouldn't be glog.V(2) or a like, like others.